frontend/src/components/pagination.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  ChevronLeft,
+  ChevronRight,
+  ChevronsLeft,
+  ChevronsRight
+} from "lucide-react";
+import { Button } from "./ui/button";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue
+} from "./ui/select";
+
+export type PaginationProps = {
+  totalPages: number;
+  currentPage: number;
+  perPage: number;
+  className?: string;
+  onChangePage: (page: number) => void;
+  onChangePerPage: (perPage: number) => void;
+};
+
+export function Pagination({
+  totalPages,
+  currentPage,
+  perPage,
+  className = "",
+  onChangePage,
+  onChangePerPage
+}: PaginationProps) {
+  return (
+    <div className={`flex items-center justify-end px-2 ${className}`}>
+      <div className="flex items-center space-x-2">
+        <p className="text-sm font-medium">Rows per page</p>
+        <Select
+          value={`${perPage}`}
```

Comment:
Does `Select` only accepts string values ? if so, you can just to `perPage.toString()` instead